frontend/javascripts/router.tsx

```diff
@@ -100,6 +100,16 @@ function PageNotFoundView() {
   );
 }
 
+function RedirectToWorkflowViewer() {
+  const location = useLocation();
+
+  useEffect(() => {
+    windowLocation.assign(`https://workflows.voxelytics.com${location.pathname}${location.search}`);
```

Comment:
The final URL will be `https://workflows.voxelytics.com/workflows/...` then, right? It's a bit redundant, but probably acceptable. Alternatively, you could trim the leading `workflows` from the pathname.

Reply:
Well the final URL will be `https://webknososs.org/workflows` ;-) I'd like to keep the URL schemes as close as possible, which introduces this (hopefully temporary) redundancy.

Reply:
ok, makes sense :+1: